src/components/Header/HeaderMenu.jsx

```diff
@@ -36,7 +36,7 @@ HeaderMenu.propTypes = {
   /** The href that the menu's label links to, wraps label in anchor if provided*/
   href: PropTypes.string,
   /** The label of the menu */
-  label: PropTypes.string.isRequired,
+  label: PropTypes.string,
```

Comment:
Do we also want to make this a `node`? Might save us from needing to do it later.

Reply:
I see. Makes sense.